src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -40,6 +40,14 @@
  * <li> 'percent'   - displays the Usage Percent of the Total amount in the center label
  * <li> 'none'      - does not display the center label
  * </ul>
+ *
+ * @param {object} lang translated i18n strings<br/>
```

Comment:
If this is an optional param, then it should be:

@param {object=} lang translated i18n strings<br/>